### PR TITLE
Quiet down testing console

### PR DESCRIPTION
**Pull request by brianfleming**
## Description

While working on [1665](https://github.com/meedan/check-web/pull/1665) and trying to fix testing. I found it kinda annoying some warnings in the output of the unit test console. It makes it more tedious to find true errors. This PR makes changes to remove some of them. The biggest chunk remaining will be cleaned up when replace the autogrow textfields with our own CDS version, since the "errors" are around the mui textarea component:
`Warning: NaN is an invalid value for the `height` css style property.`

## Type of change

- [x] Performance improvement and/or refactoring (non-breaking change that keeps existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Security mitigation or enhancement
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Automated test (add or update automated tests)

## How has this been tested?

Rerunning unit tests

## Things to pay attention to during code review

I will note in comments which errors the changes address

## Checklist

- [x] I have performed a self-review of my own code
- [ ] I've made sure my branch is runnable and given good testing steps in the PR description
- [ ] I considered secure coding practices when writing this code. Any security concerns are noted above.
- [ ] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I implemented any new components, they are self-contained, their `propTypes` are declared and they use React Hooks and, if data-fetching is required, they use Relay Modern with fragment containers
- [ ] If my components involve user interaction - specifically button, text fields, or other inputs - I have added a [BEM-like class name](https://meedan.atlassian.net/wiki/spaces/ENG/pages/1327628289/Naming+conventions+for+interactive+elements) to the element that is interacted with
- [ ] To the best of my knowledge, any new styles are applied according to the design system
- [ ] If I added a new external dependency, I included a rationale for doing so and an estimate of the change in bundle size (e.g., checked in https://bundlephobia.com/)

### CODE REVIEW

File: src/app/components/InviteNewAccount.js [hunk 1]
```diff
@@ -138,10 +138,10 @@ const InviteNewAccountComponent = ({ user }) => {
                 }}
               />
             </Typography>
-            <Typography component="div" align="center" className={classes.bold} paragraph="true">
+            <Typography component="div" align="center" className={classes.bold} paragraph>
```
Comment (brianfleming) [inline src/app/components/InviteNewAccount.js:141]: Fixes:
Screenshot 2023-10-12 at 1 18 25 PM

File: src/app/components/InviteNewAccount.js [hunk 2]
```diff
@@ -138,10 +138,10 @@ const InviteNewAccountComponent = ({ user }) => {
...
               {teamUser?.team?.name}
             </Typography>
-            <Typography component="div" align="center" paragraph="true">
+            <Typography component="div" align="center" paragraph>
```
Comment (brianfleming) [inline src/app/components/InviteNewAccount.js:144]: Fixes:
Screenshot 2023-10-12 at 1 18 25 PM

File: src/app/components/annotations/AddAnnotation.js [hunk 3]
```diff
@@ -396,18 +396,20 @@ class AddAnnotation extends Component {
...
-                size="default"
-                iconCenter={<AttachFileIcon />}
-                className={`add-annotation__insert-photo ${this.state.fileMode ? 'add-annotation__file' : ''}`}
-                buttonProps={{
-                  id: 'add-annotation__switcher',
-                }}
-                onClick={this.switchMode.bind(this)}
-              />
+            <Tooltip arrow title={<FormattedMessage id="addAnnotation.addFile" defaultMessage="Add a file" description="Tooltip to tell the user they can add files" />}>
+              <span>
```
Comment (brianfleming) [inline src/app/components/annotations/AddAnnotation.js:400]: Fixes:
Screenshot 2023-10-12 at 3 51 02 PM

File: src/app/components/cds/alerts-and-prompts/Alert.js [hunk 4]
```diff
@@ -106,8 +106,8 @@ Alert.defaultProps = {
 
 Alert.propTypes = {
   className: PropTypes.string,
-  title: PropTypes.object,
-  content: PropTypes.object,
+  title: PropTypes.oneOfType([PropTypes.object, PropTypes.string]),
```
Comment (brianfleming) [inline src/app/components/cds/alerts-and-prompts/Alert.js:109]: Fixes:
Screenshot 2023-10-12 at 3 48 34 PM

File: src/app/components/layout/LongShort.js [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components';
 import { breakWordStyles } from '../../styles/js/shared';
 
-const LongShort = styled.div`
+const LongShort = styled.p`
```
Comment (brianfleming) [inline src/app/components/layout/LongShort.js:4]: Fixes:
Screenshot 2023-10-12 at 1 23 35 PM

File: src/app/components/media/ReportDesigner/ReportDesignerFormSection.js [hunk 6]
```diff
@@ -2,15 +2,15 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { makeStyles } from '@material-ui/core/styles';
 import withStyles from '@material-ui/core/styles/withStyles';
-import ExpansionPanel from '@material-ui/core/ExpansionPanel';
-import ExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary';
-import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
+import Accordion from '@material-ui/core/Accordion';
```
Comment (brianfleming) [inline src/app/components/media/ReportDesigner/ReportDesignerFormSection.js:5]: Fixes:
Screenshot 2023-10-12 at 1 01 56 PM

File: src/app/components/search/DateRangeFilter.js [hunk 7]
```diff
@@ -418,7 +418,7 @@ const DateRangeFilter = ({
           value={getValueType()}
         >
           { ['created_at', 'media_published_at', 'updated_at', 'report_published_at', 'request_created_at'].filter(option => !optionsToHide.includes(option)).map(option => (
-            <option value={option}>{ intl.formatMessage(messages[option]) }</option>
+            <option key={option} value={option}>{ intl.formatMessage(messages[option]) }</option>
```
Comment (brianfleming) [inline src/app/components/search/DateRangeFilter.js:421]: Fixes:
Screenshot 2023-10-12 at 1 46 47 PM

File: src/app/components/search/SearchField.js [hunk 8]
```diff
@@ -158,7 +158,7 @@ const SearchField = ({
         >
           <TextArea
             disabled={!expand}
-            rows={5}
+            rows="5"
```
Comment (brianfleming) [inline src/app/components/search/SearchField.js:161]: Fixes:
Screenshot 2023-10-12 at 1 05 50 PM

File: src/app/components/team/Statuses/EditStatusDialog.js [hunk 9]
```diff
@@ -185,7 +185,7 @@ const EditStatusDialog = ({
               margin="normal"
               fullWidth
               multiline
-              rows={5}
+              rows="5"
```
Comment (brianfleming) [inline src/app/components/team/Statuses/EditStatusDialog.js:188]: Fixes:
Screenshot 2023-10-12 at 1 05 50 PM

**Comment by brianfleming** [comment]
fwiw @dariusk , seems like there are a few new ones around the Tipline history that might be quick for you to squash

File: src/app/components/cds/alerts-and-prompts/Alert.js (hunk 4 above)
Comment (amoedoamorim) [reply src/app/components/cds/alerts-and-prompts/Alert.js:109]: Perhaps this should be `PropTypes.node` instead?

**Review by amoedoamorim — APPROVED** [review]
Looks great in general, just a minor nitpick on Alert.js!

File: src/app/components/cds/alerts-and-prompts/Alert.js (hunk 4 above)
Comment (brianfleming) [reply src/app/components/cds/alerts-and-prompts/Alert.js:109]: oh yeah, good call @amoedoamorim will update

**Pull request merged**